portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,60 @@
 package com.google.sps.servlets;
 
+import java.util.ArrayList;
 import java.io.IOException;
+import com.google.gson.Gson;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
-public class HelloWorldServlet extends HttpServlet {
+public final class HelloWorldServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+    ArrayList<String> movies = new ArrayList<String>();
+    movies.add("You've Got Mail");
+    movies.add("My Big Fat Greek Wedding");
+    movies.add("Monster's University");
+
+        // Convert the server stats to JSON
```

Comment:
Make sure to keep comments up-to-date so that they always accurately describe your code.
```suggestion
    // Convert the movie list to JSON
```